src/main/resources/edu/cornell/kfs/concur/cu-spring-concur.xml

```diff
@@ -163,20 +163,141 @@
+                                    <bean parent="DelimitedFlatFilePropertySpecification"
+                                          p:lineSegmentIndex="166"
+                                          p:propertyName="subAccountNumber"/>
+                                    <bean parent="DelimitedFlatFilePropertySpecification"
+                                          p:lineSegmentIndex="167"
+                                          p:propertyName="subObjectCode"/>
+                                    <bean parent="DelimitedFlatFilePropertySpecification"
+                                          p:lineSegmentIndex="168"
+                                          p:propertyName="projectCode"/>
+                                    <bean parent="DelimitedFlatFilePropertySpecification"
+                                          p:lineSegmentIndex="169"
+                                          p:propertyName="orgRefId"/>
+                                </list>
+                            </property>
+                        </bean>
+                        <bean parent="FlatFileObjectSpecification"
+                              p:linePrefix="REQUEST ENTRY DETAIL"
+                              p:businessObjectClass="edu.cornell.kfs.concur.batch.businessobject.ConcurRequestExtractRequestEntryDetailFileLine"
+                              p:parentBusinessObjectClass="edu.cornell.kfs.concur.batch.businessobject.ConcurRequestExtractRequestDetailFileLine"
+                              p:parentTargetProperty="requestEntryDetails">
+                            <property name="parseProperties">
+                                <list>
+                                    <bean parent="DelimitedFlatFilePropertySpecification"
+                                          p:lineSegmentIndex="1"
+                                          p:formatterClass="org.kuali.rice.core.web.format.IntegerFormatter"
+                                          p:propertyName="sequenceNumber"/>
+                                    <bean parent="DelimitedFlatFilePropertySpecification"
+                                          p:lineSegmentIndex="2"
+                                          p:propertyName="requestId"/>
+                                </list>
+                            </property>
+                        </bean>
+                    </list>
+                </property>
+            </bean>
+        </property>
+        <property name="directoryPath">
+            <value>${staging.directory}/concur/requestExtract</value>
+        </property>
+        <property name="fileExtension">
+            <value>txt</value>
+        </property>
+        <property name="fileNamePrefix" value="Standard_Travel_Request_Extract_"/>
+        <property name="fileTypeIdentifier" value="concurRequestExtractInputFileType"/>
+        <property name="titleKey" value="message.batchUpload.title.achIncome"/>
```

Comment:
the title key doesn't make sense: message.batchUpload.title.achIncome. You might want to define a new one for this batch